tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "target": "es5",
+    "target": "es2015",
```

Comment:
we're in 2024, but any issues with browser compatibles going from es5? I imagine it would be minuscule

Reply:
Good question, this change might not be necessary though. I was accidentally generating the `angular` client. Not the `angularjs` client. May as well leave it the same.

Reply:
I'll do a little research though.

Reply:
Typescript docs say es6 is the way to go!

I also switched from `commonjs` to `es6`, which is recommended by webpack .